EndpointCategory/index.js

```diff
@@ -37,6 +37,7 @@ class EndpointCategory {
         this[prop] = instantiateEndpoint({
           action,
           requiredParams: endpointCoreFunction.requiredParams,
+          paramTypes: endpointCoreFunction.paramTypes,
```

Comment:
You really know your way around this code! We didn't even have to discuss how we might add `paramTypes`